go.mod

```diff
@@ -33,10 +33,10 @@ require (
 	github.com/moby/buildkit v0.0.0-20181107081847-c3a857e3fca0
 	github.com/mtrmac/gpgme v0.1.2 // indirect
 	github.com/opencontainers/go-digest v1.0.0-rc1
-	github.com/openshift/api v0.0.0-20200205133042-34f0ec8dab87
+	github.com/openshift/api v0.0.0-20200210091934-a0e53e94816b
 	github.com/openshift/build-machinery-go v0.0.0-20200211121458-5e3d6e570160
 	github.com/openshift/client-go v0.0.0-20200116152001-92a2713fa240
-	github.com/openshift/library-go v0.0.0-20200206134157-b4c763d94dcf
+	github.com/openshift/library-go v0.0.0-20200303185131-81598fff9efa
```

Comment:
This does not exist in https://github.com/openshift/library-go/tree/release-4.4/.
You need to backport the necessary library-go changes to 4.4 branch too. 
/hold

Reply:
fixed